argon-doc-viewer.css

```diff
@@ -52,68 +45,69 @@ header {
 
 /* Common Formatting */
 
+#summary {
+  padding: 4px 0;
+}
+
 #summary marked-element::shadow #content pre {
   background-color: #fafafa;
+  border: solid #ebebeb;
```

Comment:
It would be nice if `marked-element` exposed some custom properties that we could use to control these colors. No action required, just observing.